lib/util/db.js

```diff
@@ -151,15 +151,35 @@ values (${sql.join(keys.map(_assign(obj)), sql`,`)})
 returning *`;
 };
 
+// Arguments:
+// objs: An array of Frames to be inserted in the database table.
+//       Frame should defined fieldTypes for this function to work
+//       because Slonik's sql.unnest function has a mandatory columnType argument
 const insertMany = (objs) => {
   if (objs.length === 0) return sql`select true`;
   const Type = objs[0].constructor;
+  if (!Type.def.fieldTypes) throw Problem.internal.fieldTypesNotDefined(Type.from);
+
+  let columns; let rows; let columnTypes; let
+    selectExp;
+
+  // we need to set clock_timestamp if there's createdAt column
+  // Slonik doesn't support setting sql identitfier for sql.unnest yet
+  if (Type.hasCreatedAt) {
```

Comment:
this special treatment of createdAt field is due to an existing unit test, we don't have any frame with createdAt that we bulk insert today. This is nice to have for future use cases.